run_dir/design/flowcells.html

```diff
@@ -201,20 +215,38 @@ <h1><span id="page_title">Flowcells</span>
       var title = $('#fc_table thead th').eq( $(this).index() ).text();
       $(this).html( '<input size=10 type="text" placeholder="Search '+title+'" />' );
     } );
+    $('#ont_fc_table tfoot th').each( function () {
+      var title = $('#ont_fc_table thead th').eq( $(this).index() ).text();
+      $(this).html( '<input size=10 type="text" placeholder="Search '+title+'" />' );
+    } );
 
     var table = $('#fc_table').DataTable({
       "paging":false,
       "info":false,
       "order": [[ 0, "desc" ]]
     });
+    var table = $('#ont_fc_table').DataTable({
+      "paging":false,
+      "info":false,
+      "order": [[ 0, "desc" ]]
+    });
 
     //Add the bootstrap classes to the search thingy
+
+    // Illumina
     $('div.dataTables_filter input').addClass('form-control search search-query');
     $('#fc_table_filter').addClass('form-inline float-right');
```

Comment:
```suggestion
    $('#fc_table_filter').addClass('form-inline float-right pt-1');
```

Reply:
Fixed in 79872b6b752030d9478ec28321454d75a2e8f709